src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -1787,14 +1805,16 @@ Status ColorControlServer::moveSaturationCommand(EndpointId endpoint, const Comm
  * @param commandData Struct containing the parameters of the command.
  * @return Status::Success when successful,
  *         Status::UnsupportedEndpoint when the provided endpoint doesn't correspond with a saturation transition state (verified in
- * moveToSaturation function) Status::ConstraintError when a command parameters is outside its defined value range.
+ * moveToSaturation function)
```

Comment:
identation is off here here

Reply:
Styler set it this due to max char per line/end of line